flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGenerator.java

```diff
@@ -144,6 +151,18 @@ public static JobGraph createJobGraph(StreamGraph streamGraph, @Nullable JobID j
     private final StreamGraphHasher defaultStreamGraphHasher;
     private final List<StreamGraphHasher> legacyStreamGraphHashers;
 
+    private final ExecutorService serializationExecutor =
```

Comment:
Where do you shut this down?

Reply:
Thanks, I also realize it and will fix it in the next version.